Content.Client/Atmos/UI/GasThermomachineWindow.xaml.cs

```diff
@@ -17,7 +17,7 @@ public GasThermomachineWindow()
         RobustXamlLoader.Load(this);
 
         SpinboxHBox.AddChild(
-            TemperatureSpinbox = new FloatSpinBox(.1f, 2) { MaxWidth = 150, HorizontalExpand = true }
+            TemperatureSpinbox = new FloatSpinBox(.1f, 2) { MinWidth = 150, HorizontalExpand = true }
```

Comment:
This change will change the UI behavior from: (`MaxWidth`)
![dotnet_zwlgBL1Ngw](https://github.com/space-wizards/space-station-14/assets/6356337/95edab0f-d10a-4814-803f-c6659d0acf99)

To: (`MinWidth`)
![dotnet_ZdLLyguNnF](https://github.com/space-wizards/space-station-14/assets/6356337/5d286000-0f99-4146-b2ca-a61977cdedd2)


This doesn't feel intentional!

Reply:
should it even be resizable

Reply:
I changed it because i found it really annoying how the UI would cut off some numbers and make it hard to read. I can revert it but I thought it was nice.

Reply:
Fair enough!